app/services/session.js

```diff
@@ -35,6 +35,15 @@ export default Session.extend({
     return sessionId === userId;
   },
 
+  /**
+   * Get the access token.  Returns undefined when not authenticated.
+   */
+  token: computed('isAuthenticated', 'data.authenticated', function() {
+    if (!this.isAuthenticated) return;
+
+    return get(this, 'data.authenticated.access_token');
+  }),
+
```

Comment:
Am I doing something obviously wrong here @trmcnvn?  The ajax service isn't picking up on the authentication so I must be doing something wrong with computed properties :/

Reply:
Nothing jumps out at me.

Have you tried using `Ember.get` around the conditional check? Maybe an issue there with ES6 getters.

Reply:
I was thinking that but — everything works fine in a browser?

Is it possible that the test is checking before the runloop hits and updates the computed properties?

Reply:
Oh, I thought it was broken in browser unrelated to tests, my bad.

The test will be failing because the session service is mocked and `token` doesn't exist in the mock.

Reply:
Aha!  Didn't realize it was being mocked for the tests.  Thanks!

Reply:
Okay, so the issue is with `ember-auto-import` being added.

Specifically, it's a conflict with `moment/moment-timezone` as we have them included via an addon that uses shims -- and we set the timezone on authentication.

Adding the following code to `ember-cli-build.js` should fix the remaining tests:

```js
autoImport: {
  exclude: ['moment', 'moment-timezone']
}
```

However, there _may_ be other issues as well. We'll need to list all shims as an excluded import most likely.

Reply:
Well that's fun.  At least the pain should be temporary.  `ember-auto-import` is becoming default in ember-cli so shims should be going away in future